lib/std/child_process.zig

```diff
@@ -257,6 +259,55 @@ pub const ChildProcess = struct {
         }
     }
 
+    fn collectOutputWindows(child: *ChildProcess, outs: [2]*std.ArrayList(u8), max_output_bytes: usize) !void {
+        const bump_amt = 512;
+        const handles = [_]windows.HANDLE{ child.stdout.?.handle, child.stderr.?.handle };
+
+        for ([_]u1{ 0, 1 }) |i| {
+            try outs[i].ensureCapacity(bump_amt);
+            const buf = outs[i].unusedCapacitySlice();
+            child.overlapped[i] = mem.zeroes(windows.OVERLAPPED);
+            _ = windows.kernel32.ReadFile(handles[i], buf.ptr, @intCast(u32, buf.len), null, &child.overlapped[i]);
+        }
+
+        var wait_objects = handles;
+        var wait_object_count: u2 = wait_objects.len;
+
+        while (true) {
+            const status = windows.kernel32.WaitForMultipleObjects(wait_object_count, &wait_objects, 0, windows.INFINITE);
+            if (status == windows.WAIT_FAILED) {
+                switch (windows.kernel32.GetLastError()) {
+                    else => |err| return windows.unexpectedError(err),
+                }
+            }
+            if (status < windows.WAIT_OBJECT_0 or status > windows.WAIT_OBJECT_0 + wait_object_count - 1)
+                unreachable;
+
+            const wait_idx = status - windows.WAIT_OBJECT_0;
+            const i = @as(u1, if (wait_objects[wait_idx] == handles[0]) 0 else 1);
+            var read_bytes: u32 = undefined;
+            const overlapped_result = windows.kernel32.GetOverlappedResult(handles[i], &child.overlapped[i], &read_bytes, 0);
```

Comment:
This variable name is misleading as that's the overlappedresult's result, do you really need this? (no, add a trailing comma in the parameter list and move the call inside the if)

Reply:
yeah this variable is vestigal from a previous revision that did something inbetween this call and checking the result.  I've removed.